src/ui/Analytics/PendingSearchAsYouTypeSearchEvent.ts

```diff
@@ -16,6 +16,7 @@ export class PendingSearchAsYouTypeSearchEvent extends PendingSearchEvent {
   private beforeUnloadHandler: (...args: any[]) => void;
   private armBatchDelay = 50;
   private toSendRightNow: () => void;
+  private queryContent: string = '';
```

Comment:
For undefined variables, explicit typing is needed but it is not necessary here since typescript is able to infer it as `string` based on the definition.